sv_pipeline/exome/load_data.py

```diff
@@ -106,16 +107,25 @@ def _parse_genes(genes):
         'field_name': GENES_FIELD,
         'format': _parse_genes,
     },
+}
+
+NEW_JOINT_CALL_SAMPLE_COLS = {
     PREV_IDENTICAL_COL: {'field_name': 'prev_call', 'format': bool},
     PREV_OVERLAP_COL: {'field_name': 'prev_overlap', 'format': bool},
     PREV_MISSING_COL: {'field_name': NEW_CALL_FIELD, 'format': lambda call: call not in {'NA', 'FALSE'}},
 }
+
+MERGING_CALL_SAMPLE_COLS = {
+    IS_LATEST: {'field_name': 'prev_call', 'format': lambda val: not BOOL_MAP[val.strip()]},
+    PREV_OVERLAP_COL: {'field_name': 'prev_overlap', 'format': lambda val: False},
+    PREV_MISSING_COL: {'field_name': NEW_CALL_FIELD, 'format': lambda val: False},
+}
+
 COL_CONFIGS.update({col: {'format': _parse_genes} for col in GENE_CONSEQUENCE_COLS.keys()})
 
 CORE_COLUMNS = [CHR_COL, SC_COL, SF_COL, CALL_COL, IN_SILICO_COL]
 SAMPLE_COLUMNS = [
```

Comment:
the columns are the same regardless of how we do loading, so this should stay the same

Reply:
The columns do not change. The formatting functions are changed. I could update the formatting functions if there is no option of the `is_new_join_call`.